src/libAtomVM/nifs.c

```diff
@@ -3803,6 +3810,52 @@ static term nif_atomvm_read_priv(Context *ctx, int argc, term argv[])
     return result;
 }
 
+static term nif_atomvm_posix_clock_settime(Context *ctx, int argc, term argv[])
+{
+    UNUSED(argc);
+
+    VALIDATE_VALUE(argv[0], term_is_atom);
+    if (globalcontext_is_term_equal_to_atom_string(ctx->global, argv[0], ATOM_STR("\x8", "realtime"))) {
```

Comment:
@fadushin we're missing a test here that would show the logic is negated, aren't we?